src/component/AddTodoForm.js

```diff
@@ -10,31 +11,27 @@ function AddTodoForm({ onAddTodo }) {
   } 
 
   const handleAddTodo = (e) => {
-    console.log(e);
     e.preventDefault();
-    console.log(todoTitle)
     onAddTodo({id: Date.now(), title: todoTitle});
```

Comment:
There is an error when trying to add new Todo. You need to change your new Todo object to `{id: Date.now(), fields: {title: todoTitle}}`. If you take a look at the array of objects that you are getting from airtable then you will see that each object has a `fields` object attribute that contains `Title`. So, you have to use the same object format for a new record.